OneSignalSDK/unittest/src/test/java/com/test/onesignal/TestHelpers.java

```diff
@@ -373,6 +398,49 @@ else if (type == Cursor.FIELD_TYPE_FLOAT)
       return iams;
    }
 
+   @WorkerThread
+   static void deleteOldInAppMessages() {
+      SQLiteDatabase readableDb = OneSignalDbHelper.getInstance(RuntimeEnvironment.application).getReadableDatabase();
+      Cursor cursor = readableDb.query(
+              OneSignalPackagePrivateHelper.InAppMessageTable.TABLE_NAME,
+              null,
+              OneSignalPackagePrivateHelper.InAppMessageTable.COLUMN_NAME_LAST_DISPLAY + ">?",
+              new String[]{String.valueOf(SIX_MONTHS_TIME_SECONDS)},
+              null,
+              null,
+              null
+      );
+
+      if (cursor.moveToFirst()) {
+         do {
+            String idToRemove = cursor.getString(cursor.getColumnIndex(OneSignalPackagePrivateHelper.InAppMessageTable.COLUMN_NAME_MESSAGE_ID));
+            deleteInAppMessage(idToRemove);
+         } while (cursor.moveToNext());
+      }
+
+      if (!cursor.isClosed())
+         cursor.close();
+   }
+
+   static void deleteInAppMessage(String messageId) {
```

Comment:
This is duplicated, it exists in `com.onesignal.OSInAppMessageRepository` as well. Also I don't see this being called in the tests so I think we can remove this `deleteInAppMessage` method here.